packages/component-base/src/iron-list-core.js

```diff
@@ -491,9 +491,12 @@ export const ironList = {
       this._physicalIndexForKey = {};
       this._firstVisibleIndexVal = null;
       this._lastVisibleIndexVal = null;
-      this._physicalCount ||= 0;
```

Comment:
This line doesn't make sense to me, because it resets the property to 0 only if it doesn't have a value. But the getter already returns `0` in this case so apparently we don't need to re-assign it.

https://github.com/vaadin/web-components/blob/ed28a0f1f3b2a40ed26c25ffbfe915a1487b9cc3/packages/component-base/src/iron-list-core.js#L198-L200